server.js

```diff
@@ -68,15 +69,20 @@ if (typeof(routes) != "function"){
 }
 
 // auto render any view that exists
-
 app.get(/^\/([^.]+)$/, function (req, res) {
 
   var path = (req.params[0]);
 
   res.render(path, function(err, html) {
     if (err) {
-      console.log(err);
-      res.sendStatus(404);
+      res.render(path + "/index", function(err2, html) {
+        if (err2) {
+          console.log(err);
+          res.status(404).send(err).send(err2);
```

Comment:
Hey guys, I'm finding this line causes following error on my machine:
```
throw new Error('Can\'t set headers after they are sent.');
```
... because it doesn't seem to like stringing multiple `.send()` commands together.

I found this: http://stackoverflow.com/questions/25542521/cant-do-multiple-res-send-in-express-js which makes it seem like it's something that's changes from previous versions of Express.